MetadataExtractor/Extractors/Data/DicomExtract.py

```diff
@@ -0,0 +1,57 @@
+from .IDataExtract import IDataExtract
+from MetadataExtractor.Util import metadataCreation, metadataFormatter
+import pydicom
+from pathlib import Path
+from pydicom._version import __version_info__
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class DicomExtract(IDataExtract):
+    def extract(self, fileInfo):
+        log.info('Extracting metadata for Dicom file "' + fileInfo["file"] + '".')
+        text = ""
+
+        dicom_data = pydicom.dcmread(file)
+        
+        for elem in ds.iterall():
+            if elem.tag.is_private:
+                continue  # Skip private tags
+            tag_str = str(elem.tag)
+            value = elem.value
+            if isinstance(value, pydicom.valuerep.DSfloat):
+                value = float(value)
+            dicom_metadata[tag_str] = value
+
+        values = []
+        identifier = fileInfo["identifier"]
+
+        for attribute in attributes.keys():
```

Comment:
you never define any attribute, you should make use of dicom_metadata here instead of attributes